faktory/templates/server-service.yaml

```diff
@@ -15,6 +15,11 @@ spec:
       name: server
       targetPort: server
       protocol: TCP
+{{- if and (.Values.metrics.enabled) (.Values.metrics.serviceMonitor.enabled) }}
```

Comment:
Should this port be enabled on the service even if you aren't using `ServiceMonitor`? I'm not very familiar with prometheus so I'm not sure the other ways someone might run it.

Reply:
Not sure when it should be needed, but OK, let's leave it enabled for case when someone will want to write their own `ServiceMonitor`